### DISCUSSION TITLE
UI not starting

### DISCUSSION THREAD

**Discussion by themw123** [open-webui/open-webui#4480] [General]
open webui container is restarting in an endless loop. Everything worked fine until i did an update by pulling new image a few days before. Immediately after updating it did not start anymore.

i am using the command from readme:
```
docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```

docker logs open-webui:
```
Loading WEBUI_SECRET_KEY from file, not provided as an environment variable.
Generating WEBUI_SECRET_KEY
Loading WEBUI_SECRET_KEY from .webui_secret_key
USER_AGENT environment variable not set, consider setting it to identify your requests.

... repeating

```

What it tried:
 - delete all old open webui images and volumes
 - update, upgrade and restart system

System:
- Raspberry Pi 4 2gb
- Debian GNU/Linux 12 (bookworm)
- Docker version 20.10.24+dfsg1, build 297e128

**Comment by justinh-rahb** [comment]
See:
- #4331
